build-tools/scripts/BuildEverything.mk

```diff
@@ -0,0 +1,62 @@
+# $(ALL_API_LEVELS) and $(ALL_FRAMEWORKS) must be kept in sync w/ each other
+ALL_API_LEVELS    = 1 2 3 4 5 6 7 8 9 10    11  12  13  14  15      16    17    18    19    20        21    22    23    24
```

Comment:
Yes, for the same reason that `$(ALL_FRAMEWORKS)` has a bunch of `-` values.

That reason being that `$(FRAMEWORKS)` and others use the [**$(word n,text)** function](https://www.gnu.org/software/make/manual/html_node/Text-Functions.html#index-word), and **$(word)** requires 1-based values. If `$(ALL_API_LEVELS)` *doesn't* have 1..9, then `$(word 10, $(ALL_API_LEVELS)` won't be 10, which means `$(word 10, $(ALL_FRAMEWORKS)` won't be `v2.3`, which greatly hampers the ability, within make, to associate those values.